test/built-ins/DisposableStack/prototype/disposed/name.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-get-disposablestack.prototype.disposed
+description: >
+  DisposableStack.prototype.disposed.name value and descriptor.
+info: |
+  get DisposableStack.prototype.size
```

Comment:
@Ms2ger thanks for catching the wrong descriptions! They look like forgotten from copy/paste. Also this one here I guess should be
```suggestion
  get DisposableStack.prototype.dispose
```

Checking the rest of the files and pushing these changes in the next commit.

Reply:
Done